tests/datatypes/test_str.py

```diff
@@ -132,7 +139,7 @@ def test_slice(self):
 
     def test_case_changes(self):
         self.assertCodeExecution("""
-            for s in ['hello, world', 'HEllo, WORLD', 'átomo', '']:
+            for s in ['hello, world', 'HEllo, WORLD', 'tomo', '']:
```

Comment:
Sorry about that. That's what I expected but I thought that method was already implemented. I'll try to fix it.